share/revng/test/configuration/revng-qa/for-runtime-abi-tests.yml

```diff
@@ -115,14 +115,27 @@ sources:
   - tags: [for-runtime-abi-tests]
     repeat-for:
       - [x86-64-systemv-abi]
+
+      # TODO: re-enable once new PDB support is added
```

Comment:
What's this about?

Reply:
Because we don't handle PDBs this new correctly yet, these tests just fail without doing anything useful. So I disabled them for now. I left a bug-report in `~model-import`

Reply:
Status on this? Did the @dwarfyresearch patch fixes things?

Reply:
Yep, the patches look good. But they unearthed some problems I somehow missed before. Shouldn't be too hard to fix though.

Reply:
Status?

Reply:
Closing this since these are re-enabled.